.github/workflows/ci.yml

```
@@ -0,0 +1,89 @@
+name: CI
+
+on:
+  create:
+    tags:
```

actionlint (1.7.4)

"tags" filter is not available for create event. it is only for push event

(events)